spec/factories/gigs.rb

```diff
@@ -0,0 +1,27 @@
+FactoryGirl.define do
+  factory :gig do
+    date      { Faker::Date.between(Date.new(2010,7,10), 1.year.from_now) }
+    name      { proper_length_name }
+    note      { "#{Faker::Lorem.words(3)} #{Faker::Internet.url} #{Faker::Lorem.words(2)}" }
+    location  { "#{Faker::Address.city}, #{Faker::Address.state_abbr}" }
+    published false
+  end
+
+  factory :picture do
+    date      { Faker::Date.between(Date.new(2010,7,10), 1.year.from_now) }
+    name      { "[picture]" }
+    note      { "#{Faker::Internet.url}/image.png" }
+    published false
+  end
+end
+
+def between(range)
```

Comment:
I improved the comment.  This ensures that the faked dates is within a range.  The other is to ensure that faked names are not invalid due to length.